packages/cli/src/tools/loadMetroConfig.js

```diff
@@ -3,10 +3,11 @@
  * @flow
  */
 import path from 'path';
-import {createBlacklist} from 'metro';
-import {loadConfig} from 'metro-config';
-import {existsSync} from 'fs';
-import {type ConfigT} from 'types';
+import { createBlacklist } from 'metro';
```

Comment:
Please revert these. Looks like you have different Prettier settings. Running `yarn lint --fix` will be helpful

Reply:
yes sorry this is from my vscode setting. Just pushed an update after running `lint --fix` but turns out it updates `findSymlinkedModules.ts` as well. Is this intended?

Reply:
If ESLint says so, then it is intended